### PR TITLE
[docs] Update cli-usage.md to add D10 Quickstart

### PR DESCRIPTION
**Pull request by iribarne**
Added Drupal 10 Quickstart per recommendations from rfay.

## Release/Deployment notes:

<a href="https://gitpod.io/#https://github.com/drud/ddev/pull/3868"></a>

### CODE REVIEW

**Review by rfay — APPROVED** [review]
Thanks for this

**Review by heddn — APPROVED** [review]

File: docs/users/cli-usage.md [hunk 1]
```diff
@@ -196,6 +196,23 @@ ddev composer install
 ddev launch
 ```
 
+### Drupal 10 Quickstart
+
+#### Drupal 10 Composer Setup Example
+Drupal 10 is not yet released, but lots of people want to test and contribute to it, and it's easy to set it up in DDEV:
```
Comment (deviantintegral) [inline docs/users/cli-usage.md:202]:
```suggestion
[Drupal 10](https://www.drupal.org/about/10) is not yet released, but lots of people want to test and contribute to it. It's easy to set it up in DDEV:
```

**Review by deviantintegral — COMMENTED** [review]
I didn't realize there was a drupal10 template. Nice! I also like how this doesn't require PHP on the host.

This text mentions "contribute", but also doesn't actually set up a git clone you can create merge requests with. I'm unsure if that's worth going into details on. If we think the majority of readers will simply want to test various alpha / beta releases, then it's fine. But if we think readers will be checking out core issues for review, then perhaps we should link to or write up a section on how to do core development from a git clone.

**Comment by deviantintegral** [comment]
Also, if we merge this we'll want an issue to update once D10 is released? I'd hate for this text to last for years and years, but I've totally seen that happen in other projects!

File: docs/users/cli-usage.md [hunk 2]
```diff
@@ -196,6 +196,23 @@ ddev composer install
 ddev launch
 ```
 
+### Drupal 10 Quickstart
+
+#### Drupal 10 Composer Setup Example
+[Drupal 10](https://www.drupal.org/about/10) is not yet released, but lots of people want to test and contribute to it. It's easy to set it up in DDEV:
+```bash
```
Comment (rfay) [inline docs/users/cli-usage.md:203]:
```suggestion

```bash
```

File: docs/users/cli-usage.md [hunk 3]
```diff
@@ -196,6 +196,23 @@ ddev composer install
...
+cd my-drupal10-site
+ddev config --project-type=drupal10 --docroot=web --create-docroot
+ddev start
+ddev composer create --no-install drupal/recommended-project:^10@alpha
+ddev composer require drush/drush --no-install
+ddev composer install
+ddev drush site:install -y
+ddev drush uli
+ddev launch
+```
```
Comment (rfay) [inline docs/users/cli-usage.md:214]:
```suggestion
```

```

File: docs/users/cli-usage.md [hunk 4]
```diff
@@ -196,6 +196,23 @@ ddev composer install
...
+Note that as Drupal 10 moves from alpha to beta and then release, you'll want to change the tag from `^10@alpha` to `^10@beta` and then `^10`.
```
Comment (rfay) [inline docs/users/cli-usage.md:215]:
```suggestion

Note that as Drupal 10 moves from alpha to beta and then release, you'll want to change the tag from `^10@alpha` to `^10@beta` and then `^10`.

```

**Review by rfay — APPROVED** [review]
Fix minor markdown linting problems

**Pull request merged**